src/context/concrete_base.rs

```diff
@@ -96,6 +98,7 @@ impl ConcreteBaseAudioContext {
             queued_audio_listener_msgs: Mutex::new(Vec::new()),
             listener_params: None,
             offline,
+            state: AudioContextState::SUSPENDED,
```

Comment:
Our context starts running immediately, so this should be `Running`

Technically, the spec requires to start as `Suspended` and only switch to `Running` when the audio render thread is active ( https://www.w3.org/TR/webaudio/#AudioContext-constructors ), but we have not (yet) implemented that. We should probably create a ticket for that or pick it up in this PR

Reply:
I realise the OfflineAudioContext is Suspended by default, until you run `start_rendering`. So we need different behaviours

Reply:
In my first implementation, I initialized state with `Running`, but then I explored spec and found `Suspended`. I think start with `Running` is okay.
As for `OfflineAudioContext`, there is no doubt that state is `Suspended` by default and change to `Running` after `start_rendering`